geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/AbstractIngest.scala

```diff
@@ -132,8 +133,16 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
             IOUtils.closeQuietly(fw)
           }
         } catch {
-          case e: Exception =>
+          case e @ (_: ClassNotFoundException | _: NoClassDefFoundError) =>
+            val msg = s"Warning: Missing dependency for command execution: ${e.getMessage}"
+            logger.error(msg, e)
+            Command.user.error(msg)
+            sys.exit(-1)
```

Comment:
can we propagate this back up to the 'runner' class? makes me nervous having `sys.exits` deep in the call stack